Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -90,9 +93,18 @@ internal final class SettingsNotificationsViewController: UIViewController {
 
   private func animatePickerView(isHidden: Bool) {
     UIView.animate(withDuration: 0.25) {
-      self.emailPickerViewTopConstraint.constant = isHidden
-        ? 0 : -SettingsNotificationsViewController.emailPickerViewHeight
 
+      if !isHidden {
+        // Tells Voice Over to focus on the picker.
+        self.emailFrequencyPickerView.accessibilityViewIsModal = true
```

Comment:
Setting the `emailFrequencyPickerView.accessibilityViewIsModal` to `true`  forces VoiceOver to ignore other elements when the picker is shown.

Reply:
Weirdly I was able to re-focus VoiceOver back on the Email frequency cell without actually holding a reference to the cell. Instead I passed in the argument of an element that's being changed which is the picker and it worked.

If we decide to go with this solution it might be wise to test this on device with different OS versions (iOS 10/11/12).

Here's a gist of the whole function..

```
private func animatePickerView(isHidden: Bool) {
  UIView.animate(withDuration: 0.25, animations: {
    if !isHidden {
      if AppEnvironment.current.isVoiceOverRunning() {
        self.emailFrequencyPickerView.accessibilityViewIsModal = true

        UIAccessibility.post(
          notification: UIAccessibility.Notification.screenChanged,
          argument: self.emailFrequencyPickerView
        )
      }
    }

    self.emailPickerViewTopConstraint.constant = isHidden ? 0 : self.emailFrequencyPickerView.frame.height
    self.view.setNeedsLayout()
    self.view.layoutIfNeeded()
  }) { [weak self] _ in
    if isHidden {
      if AppEnvironment.current.isVoiceOverRunning() {
        // Re-enable VoiceOver focus on the table view
        self?.emailFrequencyPickerView.accessibilityViewIsModal = false

        // Notice we're passing emailFrequencyPickerView
        UIAccessibility.post(
          notification: UIAccessibility.Notification.screenChanged,
          argument: self?.emailFrequencyPickerView
        )
      }
    }
  }
}
```